starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,41 @@
 // Names and Input
-
+var hacker1 = "Inna";
+console.log(`The driver's name is ${hacker1}`);
+var hacker2 = "Sat";
+hacker2 = window.prompt("What is your name?", "hackername");
+console.log(`The navigator's name is ${hacker2}`);
 
 //Conditionals
 
+if(hacker1.length > hacker2.length){
+  console.log(`The driver has the longest name, it has ${hacker1.length} characters`);
+}
+else if(hacker2.length > hacker1.length){
+  console.log(`The navigator has the longest name, it has ${hacker2.length} characters`);
+}
+else if(hacker2.length == hacker1.length){
```

Comment:
You could've finished with just ```else``` part since you already checked all other conditions.